### PR TITLE
MES-2834 ADI  Part 2 Office Page

### PR DESCRIPTION
**Pull request by matthew2564**
## Description

Complete Office page for ADI Part 2 journey

## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2020-04-06 at 16 37 02
Screenshot 2020-04-06 at 16 38 11
Screenshot 2020-04-06 at 16 37 58

### CODE REVIEW

File: src/pages/office/cat-adi-part2/__tests__/office.cat-adi-part2.page.spec.ts [hunk 1]
```diff
@@ -42,7 +42,7 @@ import { of } from 'rxjs';
 import { MockComponent } from 'ng-mocks';
 import { RouteNumberComponent } from '../../components/route-number/route-number';
 import { CandidateDescriptionComponent } from '../../components/candidate-description/candidate-description';
-import { ShowMeQuestionComponent } from '../../components/show-me-question/show-me-question';
+// import { ShowMeQuestionComponent } from '../../components/show-me-question/show-me-question';
```
Comment (Cullima) [inline src/pages/office/cat-adi-part2/__tests__/office.cat-adi-part2.page.spec.ts]: Can we delete?

**Review by Cullima — APPROVED** [review]

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts [hunk 2]
```diff
@@ -28,11 +28,11 @@ import {
...
-import {
-  getPassCertificateNumber,
-  isProvisionalLicenseProvided,
-} from '../../../modules/tests/pass-completion/pass-completion.selector';
-import { getPassCompletion } from '../../../modules/tests/pass-completion/pass-completion.reducer';
+// import {
+//   getPassCertificateNumber,
+//   isProvisionalLicenseProvided,
+// } from '../../../modules/tests/pass-completion/pass-completion.selector';
+// import { getPassCompletion } from '../../../modules/tests/pass-completion/pass-completion.reducer';
```
Comment (tomcrawleyy) [inline src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: should this commented out code be removed?

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts [hunk 3]
```diff
@@ -53,8 +53,8 @@ interface HealthDeclarationPageState {
   candidateName$: Observable<string>;
   candidateUntitledName$: Observable<string>;
   candidateDriverNumber$: Observable<string>;
-  passCertificateNumber$: Observable<string>;
-  licenseProvided$: Observable<boolean>;
+  // passCertificateNumber$: Observable<string>;
+  // licenseProvided$: Observable<boolean>;
```
Comment (tomcrawleyy) [inline src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: same here, should this be removed?

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts [hunk 4]
```diff
@@ -147,24 +147,26 @@ export class HealthDeclarationCatADIPart2Page extends BasePageComponent {
...
-        map(isProvisionalLicenseProvided),
-      ),
+      // passCertificateNumber$: currentTest$.pipe(
+      //   select(getPassCompletion),
+      //   select(getPassCertificateNumber),
+      // ),
+      // licenseProvided$: currentTest$.pipe(
+      //   select(getPassCompletion),
+      //   map(isProvisionalLicenseProvided),
+      // ),
```
Comment (tomcrawleyy) [inline src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: same here

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts [hunk 5]
```diff
@@ -147,24 +147,26 @@ export class HealthDeclarationCatADIPart2Page extends BasePageComponent {
...
       conductedLanguage$: currentTest$.pipe(
         select(getCommunicationPreference),
         select(getConductedLanguage),
       ),
     };
 
-    const { licenseProvided$, healthDeclarationAccepted$, conductedLanguage$ } = this.pageState;
+    const {
+      // licenseProvided$,
```
Comment (tomcrawleyy) [inline src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: and here

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts (hunk 2 above)
Comment (matthew2564) [reply src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: will be removed after i rebase and your changes are in!

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts (hunk 3 above)
Comment (matthew2564) [reply src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: will be removed after i rebase and your changes are in!

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts (hunk 4 above)
Comment (matthew2564) [reply src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: will be removed after i rebase and your changes are in!

File: src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts (hunk 5 above)
Comment (matthew2564) [reply src/pages/health-declaration/cat-adi-part2/health-declaration.cat-adi-part2.page.ts]: will be removed after i rebase and your changes are in!

**Review by tomcrawleyy — APPROVED** [review]

**Pull request merged**